### ISSUE TITLE
Inception for unity

### ISSUE THREAD

**Issue by rainbon0** [tensorflow/tensorflow#27747]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using):  1.13.1

I wanted to use Tensorflow Inception in Unity with reference to this link.
 https://github.com/chen0040/unity-tensorflow-samples

Below is the code I used, but I do not see any errors, but I can not discriminate the image. In other words, nothing is displayed in the concole window. Do you know the solution?
  
first code, 
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using TensorFlow;

public class InceptionTest
{

    private TFGraph graph;
    private TFSession session;

    private static string[] labels = new string[] {
            "Linear type",
            "point type",
            "sidewalk block",
            
    };

    public void LoadModel(string your_name_graph)
    {
        if (graph == null)
        {
            Debug.Log("Loading tensor graph " + your_name_graph);
            TextAsset graphModel = Resources.Load<TextAsset>(your_name_graph);

            if (graphModel == null)
            {
                Debug.LogError("Failed to load tensor graph " + your_name_graph);
            }

            graph = new TFGraph();
            graph.Import(graphModel.bytes);
            session = new TFSession(graph);
        }
    }

    public int PredictClass(Texture2D image)
    {
        Debug.Log(image);

        float[] imageBytes = new float[image.width * image.height * 3];

        int idx = 0;
        for (int i = 0; i < image.width; i++)
        {
            for (int j = 0; j < image.height; j++)
            {
                Color pixel = image.GetPixel(i, j);

                imageBytes[idx++] = pixel.r / 255.0f;
                imageBytes[idx++] = pixel.g / 255.0f;
                imageBytes[idx++] = pixel.b / 255.0f;
            }
        }

        var runner = session.GetRunner();
        runner
            .AddInput(graph["conv2d_1_input"][0], TFTensor.FromBuffer(new TFShape(new long[] { 1, 32, 32, 3 }), imageBytes, 0, 3072))
            .AddInput(graph["dropout_1/keras_learning_phase"][0], new TFTensor(TFDataType.Bool, new long[0], 1));

        runner.Fetch(graph["output_node0"][0]);
        float[,] recurrent_tensor = runner.Run()[0].GetValue() as float[,];

        float maxVal = float.MinValue;
        int bestIdx = -1;
        for (int i = 0; i < recurrent_tensor.GetUpperBound(1); ++i)
        {
            float val = recurrent_tensor[0, i];
            if (val > maxVal)
            {
                maxVal = val;
                bestIdx = i;
            }
        }

        return bestIdx;
    }

    public string PredictLabel(Texture2D image)
    {
        int classId = PredictClass(image);
        if (classId >= 0 && classId < labels.Length)
        {
            return labels[classId];
        }
        return "unknown";
    }
}

second, 
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class cshInceptionTest : MonoBehaviour
{
    private InceptionTest classifier = new InceptionTest();
    
    // Start is called before the first frame update
    void Start()
    {
        Debug.Log(TensorFlow.TFCore.Version);
        classifier.LoadModel("tf_models/output_graph");

        string[] image_names = new string[9];
        int index = 0;
        for (int i = 1; i <= 3; ++i)
        {
            image_names[index++] = "Linear type" + i;
            image_names[index++] = "Point type" + i;
            image_names[index++] = "sidewalk block" + i;
        }
        foreach (string image_name in image_names)
        {
            Debug.Log(image_name);
            Texture2D img = Resources.Load<Texture2D>("images/inception/" + image_name);
            Debug.Log("Predicted: " + classifier.PredictLabel(img));
        }

    }

    // Update is called once per frame
    void Update()
    {
        
    }
}

First, I try to determine the image of the folder by inserting the second code into any object.
Please Help me T_T

**Comment by ymodak** [comment]
GitHub [unity-tensorflow-samples] (https://github.com/chen0040/unity-tensorflow-samples/issues) can be good platform to raise this question. Please post it on [unity-tensorflow-samples](https://github.com/chen0040/unity-tensorflow-samples/issues). Thanks!
